README.md

```diff
@@ -10,44 +10,68 @@
 [![Coverage Status](https://codecov.io/gh/pingcap/tidb/branch/master/graph/badge.svg)](https://codecov.io/gh/pingcap/tidb)
 [![GoDoc](https://img.shields.io/badge/Godoc-reference-blue.svg)](https://godoc.org/github.com/pingcap/tidb)
 
-- [**Slack Channel**](https://pingcap.com/tidbslack/)
-- **Twitter**: [@PingCAP](https://twitter.com/PingCAP)
-- [**Reddit**](https://www.reddit.com/r/TiDB/)
-- **Mailing list**: [Google Group](https://groups.google.com/forum/#!forum/tidb-user)
-- [**Blog**](https://www.pingcap.com/blog/)
-- [**For support, please contact PingCAP**](http://bit.ly/contact_us_via_github)
+<!-- vim-markdown-toc GFM -->
+
+* [What is TiDB?](#what-is-tidb)
+* [Documentation](#documentation)
+* [Quick start](#quick-start)
+* [Deployment](#deployment)
+* [Data Migration](#data-migration)
+* [Getting Help](#getting-help)
+* [Adopters](#adopters)
+* [Roadmap](#roadmap)
+* [Contributing](#contributing)
+* [License](#license)
+* [Acknowledgments](#acknowledgments)
+
+<!-- vim-markdown-toc -->
 
 ## What is TiDB?
 
 TiDB ("Ti" stands for Titanium) is an open-source NewSQL database that supports Hybrid Transactional and Analytical Processing (HTAP) workloads. It is MySQL compatible and features horizontal scalability, strong consistency, and high availability.
 
-- __Horizontal Scalability__
+For a detailed introduction to TiDB architecture, see [TiDB Architecture](https://pingcap.com/docs/dev/architecture/).
 
-    TiDB expands both SQL processing and storage by simply adding new nodes. This makes infrastructure capacity planning both easier and more cost-effective than traditional relational databases which only scale vertically.
+## Documentation
 
-- __MySQL Compatible Syntax__
+- [English](https://pingcap.com/docs)
+- [简体中文](https://pingcap.com/docs-cn)
 
-    TiDB acts like it is a MySQL 5.7 server to your applications. You can continue to use all of the existing MySQL client libraries, and in many cases, you will not need to change a single line of code in your application. Because TiDB is built from scratch, not a MySQL fork, please check out the list of [known compatibility differences](https://pingcap.com/docs/v3.0/reference/mysql-compatibility/).
+## Quick start
 
-- __Distributed Transactions with Strong Consistency__
+- [Quick start guide](https://docs.pingcap.com/zh/tidb/v4.0/quick-start-with-tidb)
```

Comment:
em, don't link the Chinese doc, or maybe we can use two versions.

Reply:
Please add two versions here.